docs/UserGuide.md

```diff
@@ -3,93 +3,128 @@
+  - **End Date**: Sort applications according to the end date of the internship application, facilitating planning and
     scheduling by highlighting approaching deadlines.
 - **User-Friendly Interface:** With a clean and intuitive design, InternBook offers ease of navigation and accessibility,
-prioritising user experience and satisfaction.
+  prioritising user experience and satisfaction.
 - **Adaptability:** Suitable for students with or without technical backgrounds, InternBook caters to a diverse user base,
-providing a seamless experience for all.
+  providing a seamless experience for all.
+
+So, whether you are just starting your internship search or you are a seasoned pro, InternBook is here to help you land
+that dream internship. Let's turn those aspirations into achievements together!
+
+Happy interning, <br>
+The InternBook Team :rocket:
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Table of Contents
+* [Getting Started](#getting-started)
+* [Features](#features)
+  * [Help](#viewing-help--help)
+  * [List](#listing-all-companies--list)
+  * [Adding Company](#adding-a-company-add)
+  * [Editing Company](#editing-a-company--edit)
+  * [Locating Company](#locating-companies-by-name-find-mag)
+  * [Deleting Company](#deleting-a-company--delete)
+  * [Marking Company](#marking-a-company--mark-heavy_check_mark)
+  * [Unmarking Company](#unmarking-a-company--unmark)
+  * [Sorting](#sorting-the-list--sort)
+  * [Setting Reminders](#setting-reminders--reminder)
+  * [Clearing All Entries](#clearing-all-entries--clear)
+  * [Exiting Program](#exiting-the-program--exit)
+  * [Saving Data](#saving-the-data)
+  * [Editing Data File](#editing-the-data-file)
+* [Future Enhancements](#future-enhancements-hammer)
+* [Frequently Asked Questions](#faq-questionquestionquestion)
+* [Known Issues](#known-issues-construction)
+* [Command Summary](#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Getting Started!
 
 > **IMPORTANT**
 > Ensure you have Java `11`  installed in your computer. If you do not have it installed, download it from [here.](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html)
+> Not sure if you have Java `11` downloaded, here's how to [check](#faq-questionquestionquestion).
 1. Download the latest `internBook.jar` from [here](https://github.com/AY2324S2-CS2103T-T13-2/tp/releases).
 
-2. Copy the file to the folder you want to use as the _home folder_ for your InternBook.
+2. Copy the file to the folder you want to use as the _home folder_ for your InternBook. 
+>:bulb: **TIP**: Create a folder with internBook.jar so that the data files created will be stored in the same folder.
 
 3. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar internBook.jar` command to run the application.
```

Comment:
Maybe can link this to the FAQ on how to launch the app, or perhaps can put it here as well as an extra information